wom/src/main/scala/wom/values/WomMap.scala

```diff
@@ -70,7 +70,7 @@ final case class WomMap private(womType: WomMapType, value: Map[WomValue, WomVal
   def tsvSerialize: Try[String] = {
     (womType.keyType, womType.valueType) match {
       case (_: WomPrimitiveType, _: WomPrimitiveType) =>
-        Success(value.map({case (k, v) => s"${k.valueString}\t${v.valueString}"}).mkString("\n"))
+        Success(value.map({case (k, v) => s"${k.valueString}\t${v.valueString}"}).mkString(start="", sep="\n", end="\n"))
```

Comment:
Per WDL 1.1, the best source we have, `If the Map is empty, an empty file is written.`, which I don't think is the case here.

Reply:
Updated. I noticed the 1.1 spec defines `write_map()` as a function for `Map[String, String]` **only**, though the code and tests here should work for all primitive types.